src/thunderbots/software/test/geom/util.cpp

```diff
@@ -264,12 +264,128 @@ TEST(GeomUtilTest, test_angle_sweep_circles)
     obs.push_back(Point(6, 8));
     obs.push_back(Point(4, 10));
 
-    testpair = angleSweepCircles(Point(0, 0), Point(10, 10), Point(-10, 10), obs, 1.0);
+    testpair = *angleSweepCircles(Point(0, 0), Point(10, 10), Point(-10, 10), obs, 1.0);
```

Comment:
You should check the optional first like you do in other tests

Reply:
changed as per above